wazo_dird/plugins/phonebook_backend/plugin.py

```diff
@@ -49,18 +53,38 @@ def load(self, dependencies: PhonebookViewDependencies):  # type: ignore[overrid
             resource_class_args=args,
         )
 
-
-class _Config(TypedDict):
-    name: str  # phonebook source name
-    tenant_uuid: str
+        def on_phonebook_delete_cascade(phonebook_uuid: str):
```

Comment:
This should happen in a db migration script and then the code should not allow this situation to happen again

Reply:
@DrPyser ?  :-)